pcsx2/gui/MainFrame.cpp

```diff
@@ -643,31 +643,65 @@ void MainEmuFrame::ApplyCoreStatus()
 		}
 	}
 
-	if( cdvd )
+	if (cdvd2 == nullptr) // It doesn't point to anything when it's explicitly destroyed , let's insert it again.
 	{
-		if( vm )
-		{
-			cdvd->SetItemLabel(_("Reboot CDVD (full)"));
-			cdvd->SetHelp(_("Hard reset of the active VM."));
-		}
-		else
-		{
-			cdvd->SetItemLabel(_("Boot CDVD (full)"));
-			cdvd->SetHelp(_("Boot the VM using the current DVD or Iso source media"));
-		}	
+		// WX throws an assert when a non stock menu item is inserted without a label text as a parameter on Insert()
+		// Let's add a placeholder text just to avoid an assert.
+		cdvd2 = m_menuSys.Insert(1, MenuId_Boot_CDVD2, "Turtleli");
```

Comment:
> Would it not be better to rearrange things so you don't need to use some random text?

Did I catch your drift ?

```
 case CDVD_SourceType::CDVDsrc_Iso:
			if (vmIsActive)
			{
				cdvd->SetItemLabel(_("Reboot ISO (full)"));
				cdvd->SetHelp(_("Hard reset of the active VM."));
               if(cdvd2)
				    cdvd2->SetItemLabel(_("Reboot ISO (fast)"));
               else
                    cdvd2 = m_menuSys.Insert(1, MenuId_Boot_CDVD2, "Reboot ISO (fast)");
				cdvd2->SetHelp(_("Reboot using fast BOOT (skips splash screens)"));
			}
```

I prefer the current code over something like the code snippet present above.